app/services/audit/event/util.rb

```diff
@@ -1,3 +1,4 @@
+# NOTE: [Rails 6] Util class is obsolete in Rails 6
```

Comment:
Going to leave it there for now, I'll slowly chip out other deprecated stuff and aim to upgrade to 6.0 defaults in the future